### ISSUE TITLE
Change schema of database

### ISSUE THREAD

**Issue by rphkhr** [kidzik/orchestration-graphs#2]
We need to store the raphaelJson in the database, to access the graph later on. I propose to change the schema of the table from:

CREATE TABLE "graphs_scenario" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "group" varchar(50) NOT NULL, "name" varchar(50) NOT NULL, "json" text NOT NULL, "date" date NOT NULL);

to 

CREATE TABLE "graphs_scenario" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "group" varchar(50) NOT NULL, "name" varchar(50) NOT NULL, "json" text NOT NULL, "raphaelJson" text NOT NULL "date" date NOT NULL);

Basically, will have a field JSON(used for Nicolas, contains activities, connections and start), and a new field RaphaelJSON (used to recreate the graph from a given scenario).

**Comment by ritterni** [comment]
Ok

**Comment by thibaultcalvayrac** [comment]
Closed by 74e6926
